CHANGES_NEXT_RELEASE

```diff
@@ -1 +1 @@
-
+- FIX: remove attribute of measures with name `id` or `type` to avoid collisions
```

Comment:
```suggestion
```

If this PR combines bump (note packages.json has been also modified) the CNR should be flushed. However, I'll keep this sentence for the future release notes in github.